src/conspiracies/corpusproc/umap_hdb.py

```diff
@@ -45,11 +45,14 @@ def filter_triplets_with_stopwords(
     """Filters triplets that contain a stopword.
 
     Args:
-        triplets (List[Tuple[str, str, str]]): List of triplets. A triplet is a tuple of three strings.
+        triplets (List[Tuple[str, str, str]]): List of triplets. A triplet is a tuple of
```

Comment:
```suggestion
        triplets: List of triplets. A triplet is a tuple of
```
Generally I would avoid adding types in the docs because you can end it with diverging types in docstrings and type hint. It should only be specified once.

(I know I have done it in some of the code beforehand)

Reply:
Good point! I actually had not noticed, since this was just code from Stine's branch that I split into two files. I'll keep an eye out for similar cases as I make my way around the repository.